tg.js

```diff
@@ -98,7 +98,7 @@ settings.experimental.header Experimental na Header
 settings.experimental.aimFreezeFix Ayusin ang Aim Freeze
 settings.experimental.mouseFlickFix Pag-ayos ng Mouse Flick
 settings.experimental.shaderRendering Pag-render
-settings.experimental.staticRendering Static Rendering
+settings.experimental.staticRendering Static na Pag-render

```

Comment:
We can do 'Static na Rendering'.

Reply:
will do, thanks

Reply:
Already committed with the others. Thanks.